<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Vault - Хранилище паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="search_layout">
      <item>
       <widget class="QLineEdit" name="search_input">
        <property name="placeholderText">
         <string>Поиск по сайту, логину или категории...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_btn">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="category_filter">
        <item>
         <property name="text">
          <string>Все категории</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_filters_btn">
        <property name="text">
         <string>Сбросить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="passwords_table"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons_layout">
      <item>
       <widget class="QPushButton" name="add_btn">
        <property name="text">
         <string>Добавить пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_btn">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate_btn">
        <property name="text">
         <string>Генератор паролей</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_btn">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
